charts/newrelic-k8s-metrics-adapter/templates/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}
+  labels:
+    {{- include "newrelic-k8s-metrics-adapter.labels" . | nindent 4 }}
+data:
+  config.yaml: |
+    accountID: {{ .Values.accountID | required "accountID is required" }}
```

Comment:
are we including them separately to perform checks?

Reply:
Also to dynamically select a value for region.